### PR TITLE
chore: Allow docletPath to be specified as a user property

### PR DESCRIPTION
**Pull request by meltsufin**

### CODE REVIEW

**Comment by lqiu96** [comment]
@meltsufin Tested with `mvn clean javadoc:aggregate -P docFX -DdocletPath=/usr/local/google/home/lawrenceqiu/IdeaProjects/java-docfx-doclet-1.7.0.jar` and using a (different) default doclet path. Both work as intended, so I believe this should work. 

I can update the monorepo kokoro script to use `-DdocletPath=${KOKORO_GFILE_DIR}/${docletName}.jar`

**Review by lqiu96 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -695,6 +695,7 @@
       <properties>
         <!-- default config values -->
         <docletName>java-docfx-doclet-1.5.0</docletName>
```
Comment (lqiu96) [inline pom.xml:697]: Latest release for doclet is v1.6.0 not sure why this wasn't ever updated

Comment (meltsufin) [reply pom.xml:697]: Let's update it in a separate PR.

Comment (lqiu96) [reply pom.xml:697]: Sure, I'll create a PR

**Pull request merged**